source/sdk/os/Env.ooc

```diff
@@ -29,7 +29,7 @@ Env: class {
 		set(key, value, true)
 	}
 	unset: static func (key: String) -> Int {
-		result := -1
+		result: Int
```

Comment:
But then you have the same problem on windows again, assigning 0 and then reassign without using it.